scripts/agency_ingestion/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+services:
+  app:
+    build:
+      context: ./
+      dockerfile: ./../Dockerfile
+      args:
+        SCRIPT_DIR: "agency_ingestion"
+    environment:
+      LOCALSTACK_URL: "http://localstack:4566"
+      MOUNTEBANK_URL: "http://mountebank:2525"
+      S3_BUCKET: "fake"
+      CDC_URL: "https://www.cdc.gov/wcms/vizdata/poxvirus/monkeypox/data/USmap_counts.csv"
+      ECDC_URL: "https://monkeypoxreport.ecdc.europa.eu"
+      WHO_URL: "https://extranet.who.int/publicemergency/api/Monkeypox/CaseSummary"
```

Comment:
Can use /api/Monkeypox for the time series instead of the aggregated counts, does CDC have timeseries data?

Reply:
For WHO I think storing daily case summaries is preferable as it will let us provide more data points. FWIW Mongo lets us make[ time series collections](https://www.mongodb.com/docs/manual/core/timeseries-collections/) as well.

Reply:
Maybe we can store both? Alternatively, I could fetch the timeseries endpoint while generating the timeseries data, that also works

Reply:
I guess that it'll be cleaner for everyone so yeah